djangocms_history/migrations/0002_auto_20170602_2002.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('djangocms_history', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='placeholderoperation',
+            name='language',
+            field=models.CharField(max_length=15, choices=[(b'en', b'en')]),
```

Comment:
Please import `django.conf import settings` over `from django.db import migrations, models`
and then set choices like so `choices=settings.LANGUAGES`.

The reason is that otherwise, django will complain about missing migration for anyone that has other languages other than english in their project.